scripts/make_gitrev.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+file=include/spl/sys/zfs_gitrev.h
+
+#
+# Set default file contents in case we bail.
+#
+rm -f $file
+echo "#define\tZFS_META_GITREV \"unknown\"" >>$file
+
+#
+# Check if we are in a git repo.
```

Comment:
...and the `git` is even installed on the system.  It's possible to be building from a tarball.